dotCMS/src/main/java/com/dotmarketing/image/focalpoint/FocalPoint.java

```diff
@@ -0,0 +1,53 @@
+package com.dotmarketing.image.focalpoint;
+
+import java.io.Serializable;
+import io.vavr.control.Try;
+
+public class FocalPoint implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+    public final float x, y;
+
+    public FocalPoint(float x, float y) {
+        this.x = x;
+        this.y = y;
+    }
+    
+    public FocalPoint(String focalPointStr) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'focalPointStr' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4761083)